build/AzurePipelinesTemplates/ProjectReunion-ProcessTestResults-Job.yml

```diff
@@ -0,0 +1,67 @@
+parameters:
+  dependsOn: ''
+  rerunPassesRequiredToAvoidFailure: 5
+  minimumExpectedTestsExecutedCount: 3000
```

Comment:
Should this be lowered while there aren't as many tests? WinUI has around 1000 tests right now being run on 5 different OS versions, so 3k tests as lower bounds is reasonable, but in this case, this will probably be not hit yet.

Reply:
yes, although test infra is not even part of the 3 phases here, but I think that's a good suggestion.  Going to lower this to 500.